src/Clap-Core.package/ClapGroup.class/instance/doc.st

```diff
@@ -0,0 +1,15 @@
+doc
+doc
+	Stdio stdout << name asUppercase.
+			Stdio stdout
+				<< ':';
+				lf;
+				crlf.
```

Comment:
Don't mix `lf` and `crlf`. For the moment you could assume we're on a unix and use `lf` only